### PR TITLE
Updated to border prop for compact table.

### PR DESCRIPTION
**Pull request by dlabaj**
Fixed border prop to show border on true and not show border on false.  #1674

### CODE REVIEW

**Comment by christiemolloy** [comment]
Can you switch the titles for the Compact Table so that the one with borders is default, and the one without borders says "Compact Table Borderless Rows"

File: packages/patternfly-4/react-table/src/components/Table/Table.d.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ export interface TableProps extends Omit<Omit<HTMLProps<HTMLTableElement>, 'onSe
   children?: ReactNode;
   className?: string;
   variant?: OneOf<typeof TableVariant, keyof typeof TableVariant>;
+  border?: boolean;
```
Comment (jschuler) [inline packages/patternfly-4/react-table/src/components/Table/Table.d.ts]: borders

Comment (dlabaj) [reply packages/patternfly-4/react-table/src/components/Table/Table.d.ts]: all set.

**Comment by mcoker** [comment]
Noticed one small thing in the prop description from the previous PR. "compacat" is misspelled and it might be better reworded. Maybe something like this?

> Indicates if borders between rows are visible on a compact table. Note that this can not be applied when using an expandable compact table.

**Pull request merged**